indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -333,8 +355,8 @@ public TaskStatus call()
                                * Users are highly suggested to be set in druid.indexer.runner.javaOpts
                                * See io.druid.concurrent.TaskThreadPriority#getThreadPriorityFromTaskPriority(int)
                                * for more information
-                              command.add("-XX:+UseThreadPriorities");
-                              command.add("-XX:ThreadPriorityPolicy=42");
+                               command.add("-XX:+UseThreadPriorities");
```

Comment:
Our IDEs needs to decide how this should go :) I think every RTR patch switches which way it wants this.